demos/ensembles/ensemble-node-classification-example.ipynb

```diff
@@ -110,118 +110,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "use_cora = True  # Select the dataset; if False, then Pubmed-Diabetes dataset is used.\n",
+    "use_cora = False  # Select the dataset; if False, then Pubmed-Diabetes dataset is used.\n",
```

Comment:
the notebook now defaults to (and shows reaults for) pubmed rather than cora - was that intentional?

Reply:
Yep, in the PR description:

> For validation, this switches the notebooks that can use either Cora or PubMedDiabetes to use PubMedDiabetes, but I'll switch back before landing.